google-cloud-datastore/src/test/java/com/google/cloud/datastore/DatastoreTest.java

```diff
@@ -1311,4 +1344,28 @@ public void testQueryWithStartCursor() {
     assertEquals(cursor2, cursor1);
     datastore.delete(entity1.getKey(), entity2.getKey(), entity3.getKey());
   }
+
+  private RunAggregationQueryResponse dummyAggregationQueryResponse() {
```

Comment:
rename to `placeholderAggregationQueryResponse`

Reply:
Fixed as part of https://github.com/googleapis/java-datastore/pull/823/commits/a5b462332db2f8b88db4016d2f83046c89e74acd.